content/blog/kubernetes-cluster-as-code-part-2.md

```diff
@@ -0,0 +1,347 @@
+      + timeout                    = 300
+      + verify                     = false
+      + version                    = "36.0.2"
+      + wait                       = true
+      + wait_for_jobs              = false
+ 
+      + set {
+          + name  = "grafana.service.type"
+          + value = "NodePort"
+        }
+    }
+ 
+Plan: 1 to add, 0 to change, 0 to destroy.
+ 
+────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+ 
+Note: You didn't use the -out option to save this plan, so Terraform can't guarantee to take exactly these actions if you run "terraform apply" now.
+```
+
+<!--StartFragment-->
+
+### Terraform ready to apply
+
+Terraform apply executes the actions proposed in the Terraform plan and deploys the resources. Run **terraform apply** and then type yes when asked to **Enter a value**.
+
+<!--EndFragment-->
+
+```markdown
+$ terraform apply
+ 
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+  + create
+ 
+Terraform will perform the following actions:
+ 
+  # helm_release.prometheus-stack will be created
+  + resource "helm_release" "prometheus-stack" {
+      + atomic                     = false
+      + chart                      = "kube-prometheus-stack"
+      + cleanup_on_fail            = false
+      + create_namespace           = false
+      + dependency_update          = false
+      + disable_crd_hooks          = false
+      + disable_openapi_validation = false
+      + disable_webhooks           = false
+      + force_update               = false
+      + id                         = (known after apply)
+      + lint                       = false
+      + manifest                   = (known after apply)
+      + max_history                = 0
+      + metadata                   = (known after apply)
+      + name                       = "prometheus-stack"
+      + namespace                  = "test-namespace"
+      + recreate_pods              = false
+      + render_subchart_notes      = true
+      + replace                    = false
+      + repository                 = "https://prometheus-community.github.io/helm-charts"
+      + reset_values               = false
+      + reuse_values               = false
+      + skip_crds                  = false
+      + status                     = "deployed"
+      + timeout                    = 300
+      + verify                     = false
+      + version                    = "36.0.2"
+      + wait                       = true
+      + wait_for_jobs              = false
+ 
+      + set {
+          + name  = "grafana.service.type"
+          + value = "NodePort"
+        }
+    }
+ 
+Plan: 1 to add, 0 to change, 0 to destroy.
+ 
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
```

Comment:
Suggest rephrasing this to:
" 'Yes" is the only accepted response that can be used to confirm the action."

Reply:
This is a terraform generated output on running "terraform apply". Hence, leaving it unchanged.